docs/asyncapi.yaml

```diff
@@ -214,50 +205,100 @@ components:
         name:
           type: string
           description: The funding option name
-          example: Arable and horticultural soils
+          example: Total
         level:
           type: string
           description: The funding option level
-          example: Intermediate
+          example: ''
         area:
           type: string
           description: The funding option area in hectares
-          example: 10.00
+          example: '45.4545'
         rate:
           type: string
           description: The funding option rate in pounds per hectare
-          example: 40.00
+          example: ''
         annualValue:
           type: string
           description: The funding option annual value in pounds
-          example: 400.00
+          example: '1000.00'
         quarterlyValue:
           type: string
           description: The funding option quarterly value in pounds
-          example: 100.00
+          example: '250.00'
         quarterlyReduction:
           type: string
           description: The funding option quarterly reduction in pounds
-          example: 75.00
+          example: '0.00'
         quarterlyPayment:
           type: string
           description: The funding option quarterly payment in pounds
-          example: 25.00
-        reductions:
-          type: array
-          items:
-            $ref: '#/components/schemas/Reduction'
-    Reduction:
+          example: '250.00'
+    Payment:
       type: object
       required:
-      - reason
+      - invoiceNumber
+      - reference
+      - dueDate
+      - settled
+      - calculated
       - value
+      - period
       properties:
-        reason:
+        invoiceNumber:
```

Comment:
These should all be schemas and referenced

Reply:
What should all be schemas and referenced?

Reply:
See previous comment